definitions/npm/bull_v1.x.x/flow_>=v0.22.x/bull_v1.x.x.js

```diff
@@ -0,0 +1,56 @@
+declare type $npm$bull$jobType =
```

Comment:
Okay, that's new... why prefix your classes with `$npm$bull` instead of just `bull$`? :-)